common.h

```diff
@@ -390,18 +390,19 @@ typedef int blasint;
 #define YIELDING
 #endif
 
+#if defined(_MSC_VER) && !defined(__clang__)
+#undef YIELDING // MSVC doesn't support assembly code
+#define YIELDING    	YieldProcessor()
+#endif
+
 #ifndef YIELDING
 #if defined(OS_SUNOS)
 #define YIELDING	thr_yield()
 
 #elif defined(OS_WINDOWS)
-# if defined(_MSC_VER) && !defined(__clang__)
-# define YIELDING    	YieldProcessor()
-# else
-# define YIELDING	SwitchToThread()
-# endif
+#define YIELDING	SwitchToThread()
 
-#else
+#else // assume linux
```

Comment:
not only linux but *bsd as well - and probably other unixoid systems as well - this is just "assume sched_yield may work better with an unknown scheduler than doing nothing at all", and was historically what GotoBLAS/OpenBLAS would use on any host. perhaps simply leave out the comment ?